src/main/java/com/shippingcostestimator/enterprise/PackageEstimatorController.java

```diff
@@ -20,7 +20,7 @@
 
     /**
      * Handle the root ("/") endpoint and return a start page.
-     * @return
+     * @return start.html page.
      */
```

Comment:
Added return description for JavaDoc. [Ref](https://medium.com/@rhamedy/a-short-summary-of-java-coding-best-practices-31283d0167d3)